README.md

```diff
@@ -1,4 +1,3 @@
-[![Build Status](https://travis-ci.com/PhoenicisOrg/scripts.svg?branch=master)](https://travis-ci.com/PhoenicisOrg/scripts)
```

Comment:
Should we add a GitHub Actions Badge instead?

Reply:
Question is: for which one?